hack/verify-gofmt.sh

```diff
@@ -0,0 +1,33 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+GO_VERSION=($(go version))
+
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.10|go1.11|go1.12') ]]; then
+  echo "Unknown go version '${GO_VERSION[2]}', skipping gofmt."
+  exit 1
+fi
+cd "${WMCO_ROOT}"
+find_files() {
```

Comment:
Please add a comment as to what this macro is doing. Also move it to a separate spot in the file so that it makes it easier to read the flow.